config/locales/views/submissions/en.yml

```diff
@@ -61,6 +61,13 @@ en:
       not_collected: Not Collected
       released: Released
       remark_requested: Remark Requested
+    state_filter:
```

Comment:
Instead of defining new keys here that are essentially the same text as the `state:` keys (above), just use those ones and just append the count to the end of the string. That way we don't have to maintain two sets of translations that are essentially the same.

So instead of calling:

```
I18n.t("submissions.state_filter.in_progress", {count: marking_states["incomplete"]})
```

do something like:

```
I18n.t("submissions.state.in_progress") + ` (${marking_states["incomplete"]})`
```

Reply:
Makes sense! Done